### ISSUE TITLE
NTR: Myositis-associated autoantibodies (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8750]
New term request Myositis-associated autoantibodies (hausma_120830192615): MAA, anti-p155, anti-MJ, myositis-specific autoantibodies, MSA; Anti-synthetase antibodies

**Comment by pnrobinson** [comment]
@MickeySegal

We can add specific Abs later on


Adding new term: 
- ID: HP:6000678
- label: Myositis-specific autoantibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:26424665
- def: The presence of autoantibodies (immunoglobulins) in the serum that display a high degree of specificity for idiopathic inflammatory myopathy (myositis-specific autoantibodies (MSAs)). MSAs are clinically useful biomarkers to help the diagnosis of polymyositis/dermatomyositis (PM/DM). Many of these are also associated with a unique clinical subset of PM/DM, making them useful in predicting and monitoring certain clinical manifestations.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8750
- synonyms:
